### PR TITLE
Create Worshipper

### PR DESCRIPTION
**Pull request by RogueAkira**
Worshipper role suggestion

### CODE REVIEW

**Review by Topkinsme — CHANGES_REQUESTED** [review]
The template is a bit off. You have mentioned some things which I feel like should be in the description of the pr itself.

Regarding the role itself, it's a decent idea, but might be too op. The insta respawn and 4 phase reduction along with the extension for the enemies could be brutal. However, I still think it could work.

Just fix the formatting when you can tho. Thanks!

**Comment by RogueAkira** [comment]
> The template is a bit off. You have mentioned some things which I feel like should be in the description of the pr itself.
> 
> Regarding the role itself, it's a decent idea, but might be too op. The insta respawn and 4 phase reduction along with the extension for the enemies could be brutal. However, I still think it could work.
> 
> Just fix the formatting when you can tho. Thanks!

Thanks for your reply! I'm new to github and I will do the necessary things that are required...

**Review by Topkinsme — COMMENTED** [review]
Why is everything in quotes what

**Comment by TheEighthDeadlySin** [comment]
I have a few questions about this role:
1. Will the prayers that require 2 phases to cast fail if the Worshipper dies during that time?
2. What happens if you want to extend someone's respawn time but they respawn right when you finish your incantation?
3. What will happen if the last King dies just as you finish your incantation to reduce your teammate's respawn time so that they instantly respawn?
4. Is each prayer only available once per game? If not, what are (if there are any) the cooldowns for each type of prayer?
5. If each prayer is a one time use, can you use multiple prayers at the same time?
6. If each prayer is a one time use and a specific prayer fails due to death or some other reason, will you lose that prayer?

**Comment by RogueAkira** [comment]
> I have a few questions about this role:
> 
> 1. Will the prayers that require 2 phases to cast fail if the Worshipper dies during that time?
> 2. What happens if you want to extend someone's respawn time but they respawn right when you finish your incantation?
> 3. What will happen if the last King dies just as you finish your incantation to reduce your teammate's respawn time so that they instantly respawn?
> 4. Is each prayer only available once per game? If not, what are (if there are any) the cooldowns for each type of prayer?
> 5. If each prayer is a one time use, can you use multiple prayers at the same time?
> 6. If each prayer is a one time use and a specific prayer fails due to death or some other reason, will you lose that prayer?

The role Worshiper is similar to that of a Barricader but they alter the respawn phases instead of protecting.
Answers to your queries by order:
1.Yes the prayer fails and would be on hold and after they come back alive they could use it
2.Their respawn time increases and they will not respawn that phase
3.They instantly respawn
Question number 4,5 has been answered by the starting Barricader similarity sentence
6.Nope since the prayers are on hold and will finish once they come back alive 
Thanks for your time!

**Comment by RogueAkira** [comment]
> Why is everything in quotes what

Dude u told me to do "-"

**Comment by Topkinsme** [comment]
> > Why is everything in quotes what
> 
> Dude u told me to do "-"

I asked you to use hyphens (-) in front of the point, not quotes

**Comment by 12Metatron** [comment]
should have a cooldown on each power, like mage

File: respawn_altering/Worshipper [hunk 1]
```diff
@@ -1,8 +1,7 @@
...
-2.Earthly Incantation- could reduce respawn phases by 4 for the team members who are respawning.
-3.Demonic Incantation-could increase respawn phases by 2 for a enemy .
-Each prayer takes 2 phases to cast except the Heavenly incantation which is immediate upon use
-Respawns in 6phases
-This role is similar to that of a barricader(3 prayers which resets upon death)and Wizard(Respawn alteration)
+Worshipper (7)
+"The Worshipper has the ability to use 3 types of prayers."
+"1.Heavenly incantation- could respawn a person immediately."
+"2.Earthly Incantation- could reduce respawn phases by 4 for the team members who are respawning."
+"3.Demonic Incantation-could increase respawn phases by 2 for a enemy."
```
Comment (12Metatron) [inline respawn_altering/Worshipper:5]: spacing inconsistency here.

**Comment by 12Metatron** [comment]
2 small things.

1. the role files seems to always be lowercase, this one is upper case
2. needs to say what type of action this is (for action order purposes), in its own line near the bottom like the other roles. Personally, I suggest you say it's cast as a post-night action, which is what the the other respawn altering roles have it as

**Review by RogueAkira — COMMENTED** [review]
Changed some grammar and reworded some descriptions for ease of understanding.

**Review by Topkinsme — APPROVED** [review]
Still good, power levels open to debate

**Pull request merged**